ice-adapter/src/main/java/com/faforever/iceadapter/ice/PeerIceModule.java

```diff
@@ -153,11 +158,50 @@ private void gatherCandidates() {
     //How often have we been waiting for a response to local candidates/offer
     private volatile int awaitingCandidatesEventId = 0;
 
+    private List<IceServer> getViableIceServers() {
```

Comment:
All ice servers are viable as we already added them all for stun. I'd rename to `getViableRelay(s/Servers)`